README.md

```diff
@@ -119,22 +119,26 @@ repository](https://github.com/open-telemetry/community/blob/main/community-memb
 
 ## Release Schedule
 
-OpenTelemetry .NET is under active development.
-
-The library is not yet _generally available_, and releases aren't guaranteed to
-conform to a specific version of the specification. Future releases will not
-attempt to maintain backwards compatibility with previous releases. Each alpha
-and beta release includes significant changes to the API and SDK packages,
-making them incompatible with each other.
+Only the [core components](./VERSIONING.md#core-components) of the repo has
+released a stable version. These components have a non pre-release version of
```

Comment:
"These components have a non pre-release version of..." is a bit hard to understand, and it is technically not part of "release schedule", perhaps we just keep it short and point people to the VERSIONING doc?